nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/logging/repository/StandardLogRepository.java

```diff
@@ -34,7 +34,8 @@
 
 public class StandardLogRepository implements LogRepository {
 
-    public static final int DEFAULT_MAX_CAPACITY_PER_LEVEL = 10;
+    public static final String NEW_LINE_ARROW = "\u2517\u2501\u25B6";
```

Comment:
We could use a dedicated unicode character instead.
```suggestion
    public static final String NEW_LINE_ARROW = "\u21B3";
```

Reply:
I tried thank you, this one looks to small.